doc/mgmt/gnmi/SONiC_GNMI_Server_Interface_Design.md

```diff
@@ -164,7 +164,12 @@ The ultimate goal is to use SONiC Yang models for configuration, and we also nee
 
 * We will follow OpenConfig specification to use origin field to support mixed schema.
 
-* And we use the first element to specify the target database.
+* We use the first element to specify the target database.
+
+* We use the second element to specify the database instance:
+  * For single asic device, this element is "localhost"
```

Comment:
Does single asic recognize "localhost" and parse that from GNMI's end?
I ask this because when we show running config, multi-asic has one more asic layer than single asic.

multi-asic's running config:
```
{
  "localhost": {// host config
    "ACL_TABLE":{..}
  },
  "aisc0": {// asic0 config, only exists if multi-asic
    "ACL_TABLE":{..}
  }
}
```
single asic's running config:
```
{//host's config
  "ACL_TABLE":{..}
}
```